sv_inline.h

```diff
@@ -102,7 +105,7 @@ S_new_SV(pTHX_ const char *file, int line, const char *func)
 
     return sv;
 }
-#  define new_SV(p) (p)=S_new_SV(aTHX_ __FILE__, __LINE__, FUNCTION__)
+#  define new_SV(p) (p)=S_new_SV(aTHX_ __FILE__, __LINE__, __FUNCTION__)
```

Comment:
`FUNCTION__` was correct. See `handy.h`: "Try to figure out \_\_func\_\_ or \_\_FUNCTION\_\_ equivalent, if any."

Reply:
Thanks, so it was!